### ISSUE TITLE
Refactor src/ to something else

### ISSUE THREAD

**Issue by mirzahilmi** [bccfilkom-be/Workshop-Intern-2024#4]
## TL;DR
`src` directory is not common pattern nor encouraged in Go projects, change it to something else.

## Details
Some Go projects do have a src folder, but it usually happens when the devs came from the Java world where it's a common pattern. It less or more discouraged pattern in Go projects because it can confuse project-level `src` with Go workspaces `src` and end up being a subdirectory of it. For more explanation why this become an issue particularly in Go projects, you can read it [here](https://github.com/golang-standards/project-layout?tab=readme-ov-file#src).  

![go-src-issue](https://github.com/bccfilkom-be/Workshop-Intern-2024/assets/118604764/0968c1e8-bb4f-4d3c-95b2-57696c122e1d)

## Solution
Refactor `src` directory to something else. My own preference would be the `internal` directory but that up to the maintainer and others consideration, im here to give suggestion and my personal opinion. 

Regards,
Mirza
